Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -170,12 +171,14 @@ function Main {
 				Write-LogInfo "VM is still resuming!"
 			}
 		}
+
 		if ($vmCount -le 0){
 			Write-LogInfo "VM resume completed"
+			break
```

Comment:
break? this statement is outside loop.

Reply:
fixed in b1e37ec3b2cff5abd8056db8c8d375d4a1674af5